public/app/plugins/datasource/graphite/datasource.test.ts

```diff
@@ -128,9 +130,12 @@ describe('graphiteDatasource', () => {
         });
       });
 
-      await ctx.ds.query(query as any).then((data: any) => {
-        results = data;
-      });
+      await ctx.ds
+        .query(query as any)
+        .toPromise()
+        .then((data: any) => {
+          results = data;
+        });
```

Comment:
This looks a bit weird with an `await` and a Promise chain? And shouldn't we test the Observable not the Promise?